lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -408,8 +408,8 @@ protected function _getPortableTableForeignKeysList($tableForeignKeys)
         $list = [];
         foreach ($tableForeignKeys as $value) {
             $value = array_change_key_case($value, CASE_LOWER);
-            $name  = $value['constraint_name'];
-            if (! isset($list[$name])) {
+            $index = $value['id'];
```

Comment:
Since the array key is no longer a combination of the name and id, would it make sense to rename the variable to `$id` in order to not introduce new terms?